users/services/login.py

```diff
@@ -0,0 +1,23 @@
+from django.contrib.auth.models import User
+from django.core.exceptions import ObjectDoesNotExist
+from one_day_intern.exceptions import InvalidLoginCredentialsException
+from . import utils
+
+
+def verify_password(user: User, password: str):
+    if not user.check_password(password):
```

Comment:
Hello, Jo. If I may ask, is there a builtin validator for password validation? If there is one, I think it might be wiser to use it.